docs/source/podman.md

```diff
@@ -45,7 +47,9 @@ Choose one from the following list:
 
 **optional:scale** REDIS=$REDIS [For Vdbench only: redis for scale synchronization]
 
-**optional:** LSO_PATH=$LSO_PATH [LSO_PATH='/dev/sdb/' For hammerdb only: for using Local Storage Operator]
+**optional:** LSO_DISK_ID=$LSO_DISK_ID [LSO_DISK_ID='scsi-3600062b33333333333333333333333333' For using Local Storage Operator in hammerdb]
```

Comment:
Unless that name is an actual literal, do something like

```
scsi-<replace_this_with_your_actual_disk_id>
```
(that is the literal text, I don't suggest putting a real ID in there).